src/main/groovy/org/standardout/gradle/plugin/platform/internal/FileBundleArtifact.groovy

```diff
@@ -134,13 +134,13 @@ class FileBundleArtifact implements BundleArtifact {
 	 */
 	FileBundleArtifact(BundleArtifact bundle, File sourceBundleFile) {
 		this.file = sourceBundleFile
-		this.id = sourceBundleFile as String
+		this.id = bundle.id + ':sources'
 		this.source = true
 		this.wrap = false // wrapping is done implicitly
 		
 		bndConfig = null
 		
-		version = modifiedVersion = bundle.version
+		version = modifiedVersion = VersionUtil.toOsgiVersion(bundle.version).toString()
```

Comment:
Ah ok. That was actually my first approach, but then I wasn't sure if `modifiedVersion` is always available even if the version isn't being modified. If it is, that's of course the better approach.